docker-compose.yml

```
@@ -50,6 +50,12 @@ services:
       - ./:/go/src/app
       - ./.gocache:/go/pkg/mod
 
+  pact-publisher:
+    image: pactfoundation/pact-cli:1
+    working_dir: /go/src/app
+    volumes:
+      - ./:/go/src/app
```

I think we only need to mount the pacts folder?
```suggestion
    volumes:
      - ./pacts:/go/src/app/pacts
```